portal-impl/src/com/liferay/portlet/asset/service/impl/AssetTagLocalServiceImpl.java

```diff
@@ -122,7 +122,7 @@ public AssetTag addTag(
 		List<AssetTag> tags = new ArrayList<>();
 
 		for (String name : names) {
-			AssetTag tag = fetchTag(group.getGroupId(), name);
+			AssetTag tag = fetchTag(group.getGroupId(),  StringUtil.toLowerCase(StringUtil.trim(name)));
```

Comment:
how about the `name`  variable here?

Reply:
at line 134

Reply:
Code can run without error on that case, however on the clean code aspect. This is not good enough.
Because when another team member review this method they will wonder why the name variable must be lowercase but the other place is not.

Reply:
I understand. Thank you.